packages/manager/modules/advices/src/advices/component.js

```diff
@@ -91,6 +91,11 @@ export default {
      */
     adviceType: '@',
 
+    /**
+     * Type of recommender system advice. Available values are ['baremetal', 'domain', 'public_cloud', 'saas', 'vps', 'web']
```

Comment:
**nitpick:** Can you please complete the list?
I see `telephony` in the file packages/manager/apps/telecom/src/app/telecom/telephony/line/management/management.html which is not present in the list

Reply:
Right, missed